Add SPI acquisition for X-Heep

FrancescoPoluzzi opened the pull request:
- Added SPI acquisition in X-Heep
- updated README including X-Heep

FrancescoPoluzzi committed:
Add SPI acquisition for X-Heep

- Added SPI acquisition in X-Heep
- updated README including X-Heep

davideschiavone commented on Applications/Spi_Acquisition/README.md:
```diff
@@ -0,0 +1,10 @@
+# SPI acquisition for X-Heep MCU Board
+
+This application reads a 768-Bytes buffer from a SPI ADC device. Macros are avaliable for debugging and profiling.
```
please write that one can optimize it (but do not do it) by using the DMA

FrancescoPoluzzi committed:
Update README

specified in the README the possibilty of optimizations trough DMA and power manager

davideschiavone commented:
LGTM, I let @DimitriosSamakovlis take care of extra revision and merge

DimitriosSamakovlis commented:
Overall looks good.

For consistency, please:
1. add the license + author/date structure at the top of your main.c (easy copy-paste from other files - put your name as main author)
2. move the Readme inside the x-Heep folder

Then you are all done!

FrancescoPoluzzi committed:
fix license, reorder files
